packages/js/src/shared-admin/components/form/form-layout.js

```diff
@@ -0,0 +1,91 @@
+import { useCallback, useMemo } from "@wordpress/element";
+import { Button, useToggleState } from "@yoast/ui-library";
+import { Form, useFormikContext } from "formik";
+import { includes, values } from "lodash";
+import PropTypes from "prop-types";
+import AnimateHeight from "react-animate-height";
+import { UnsavedChangesModal } from "../unsaved-changes-modal";
+
+/**
+ * @typedef {Object} FormLayoutProps
+ * @property {React.ReactNode} children
+ * @property {boolean} [isExternalLoading]
+ * @property {Function} [onUndo]
+ */
+
+/**
+ *  Form Layout with save and cancel buttons
+ *
+ * @param {FormLayoutProps} props
+ * @returns {JSX.Element}
+ */
+export const FormLayout = ( {
+	children,
+	isExternalLoading = false,
+	onUndo = () => {},
+} ) => {
```

Comment:
We usually don't use it for props of a component if we deconstruct it, we usually add it directly to the js docs. So I ment like that ( for children we use type `JSX.node`:
```js

/**
 *  Form Layout with save and cancel buttons
 *
 * @param {JSX.node} children
 * @param {boolean} [isExternalLoading=false] 
 * @param {Function} [onUndo=noop]
 * 
 * @returns {JSX.Element}
 */
export const FormLayout = ( {
	children,
	isExternalLoading = false,
	onUndo = () => {},
} ) => {
```